spec/models/thredded/post_spec.rb

```diff
@@ -69,7 +65,7 @@ module Thredded
       topic = create(:topic)
       future_time = 3.hours.from_now
       create(:post, postable: topic, user: joel, content: 'posting here')
-      Timecop.freeze(future_time) do
+      travel_to future_time do
```

Comment:
Will need to remove Timecop entirely and use the built-in rails helper. This is just a start.